sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.schema.yaml

```diff
@@ -66,3 +66,6 @@ fields:
 - mode: NULLABLE
   name: install_source
   type: STRING
+- mode: NULLABLE
+  name: days_active_dau_bits
```

Comment:
Might be worth including a `description` for this field for clarity on how it is different than the `days_active_bits` field.

Reply:
@irrationalagent another question about naming, my impression was that generally we wanted to avoid directly relating things to `dau`, is this name fine or should we consider calling it something different?

Reply:
> See my comment above, maybe we are ok with "overriding" the current days_active_bits field for desktop, however that would be a breaking change.

I agree it would be a breaking change. Also, we would be overloading the definitions. I am inclined towards keeping them separate.

For naming what about `days_desktop_active_bits` to indicate its mainly for desktop computations only?

Reply:
OK `days_desktop_active_bits` is fine by me.